src/processor/mapper/map_ddl.cpp

```diff
@@ -53,6 +55,15 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapLogicalCreateRelTableToPhysical
         &storageManager.getRelsStore().getRelsStatistics());
 }
 
+std::unique_ptr<PhysicalOperator> PlanMapper::mapLogicalCopyToToPhysical(
+    LogicalOperator* logicalOperator) {
+    auto copy = (LogicalCopyTo*)logicalOperator;
+    auto tableName = catalog->getReadOnlyVersion()->getTableName(copy->getTableID());
+    // stuff...
```

Comment:
The goal of mapper is to construct `vector<DataPos> vectorsToCopyPos`. You can do so by 
```
auto childSchema = logicalOperator.getChild(0).getSchema();
vector<DataPos> vectorsToCopyPos;
for (auto& expression : boundCopy.getCopyToInfo.expressionsToCopy ) {
     vectorsToCopyPos.emplace_back(childSchema.getExpressionPos(expression))
}
```